charts/flyte-core/templates/admin/deployment.yaml

```diff
@@ -17,6 +17,10 @@ spec:
         {{- end }}
       labels: {{ include "flyteadmin.labels" . | nindent 8 }}
     spec:
+      securityContext:
+        fsGroup: 1001
+        runAsUser: 1001
+        fsGroupChangePolicy: "OnRootMismatch"
       initContainers:
         {{- if .Values.db.checks }}
         - name: check-db-ready
```

Comment:
I dont have a easy way of testing how this user definition works together with the initContainer defined below.

Reply:
I think the best way to do this is to:

```bash
flytectl sandbox start
helm list -a -A
helm uninstall -n flyte <helm chart name>
helm install flyte --debug ./charts/flyte -f ./charts/flyte/values-sandbox.yaml --create-namespace --namespace=flyte
```

and see what happens :-)